src/utils/__tests__/transformToHarRequest.test.js

```diff
@@ -0,0 +1,79 @@
+import { transformToHarRequest } from '../transformToHarRequest';
+
+describe('transformToHarRequest', () => {
+  it('transforms GET request', () => {
+    const data = {
+      method: 'GET',
+      path: '/example',
+      headers: {
+        'X-Api-Key': 'api-key',
+      },
+    };
+
+    const expected = {
+      method: 'GET',
+      url: 'https://service.centrapay.com/example',
+      httpVersion: 'HTTP/2.0',
+      headers: [
+        {
+          name: 'X-Api-Key',
+          value: 'api-key'
+        }
+      ],
+    };
+    const expectedString = JSON.stringify(expected, null, 2); // Pretty print JSON
+
+    expect(transformToHarRequest(data)).toBe(expectedString);
+  });
+
+  it('transforms post data to HAR request', () => {
+    const data = {
+      method: 'POST',
+      path: '/example',
+      headers: {
+        'X-Api-Key': 'api-key',
+        'Content-Type': 'application/json',
+      },
+      postData: { key: 'value' },
+    };
+
+    const expected = {
+      method: 'POST',
+      url: 'https://service.centrapay.com/example',
+      httpVersion: 'HTTP/2.0',
+      headers: [
+        {
+          name: 'X-Api-Key',
+          value: 'api-key'
+        },
+        {
+          name: 'Content-Type',
+          value: 'application/json'
+        }
+      ],
+      postData: {
+        mimeType: 'application/json',
+        text: '{\"key\":\"value\"}',
+      }
+    };
+    const expectedString = JSON.stringify(expected, null, 2); // Pretty print JSON
+
+    expect(transformToHarRequest(data)).toBe(expectedString);
+  });
+
+  it('does not require headers', () => {
```

Comment:
I'm not sure whether we should require headers or not. All of our requests in our docs tend to have an auth header.

Reply:
Resolution: Discussed this and have decided to leave as is seeing as we do have some public endpoints that could be documented with no headers.